huaweicloud/resource_huaweicloud_networking_eip_associate.go

```diff
@@ -1,139 +0,0 @@
-package huaweicloud
-
-import (
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-
-	"github.com/chnsz/golangsdk"
-	"github.com/chnsz/golangsdk/openstack/networking/v2/extensions/layer3/floatingips"
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
-)
-
-// ResourceNetworkingFloatingIPAssociateV2 is not uesd in huaweicloud,
-// but it is used by other Joint-Operation cloud providers, so keep it.
```

Comment:
Is that safe to remove this? it is used by other Joint-Operation cloud providers as the comments said above.

Reply:
this comments was introduced by #1908. Now Joint-Operation cloud providers can upgradate the vendor version, and use `eip.ResourceEIPAssociate` to implement the resource.